client/src/components/layout.tsx

```diff
@@ -31,6 +34,7 @@ const Layout = ({children, location}: ILayoutProps) => {
           {conditionalAside}
         </Grid>
       </GridContainer>
+      {isMethodologyPage ? <DatasetContainer />: null}
```

Comment:
It is `null` or `<></>` or `< />` ? @NatHillardUSDS (is this a style thing?)

Reply:
Good call out! 

TL:DR
It seems the first link says, `< />` creates a React Fragment which might be more of a lift than null. Then it said the TS didn't handle JSX elements returning null. Then the 2nd link shows how the official TS issue from MS allows for null now.

1. https://www.reddit.com/r/reactjs/comments/gtyr61/null_vs_empty_fragment/  , which lead to this:
2. https://github.com/microsoft/TypeScript/issues/11566

Maybe we should make null the standard on conditional renders that return nothing?

Reply:
Thanks for doing the research. I'm happy with using `null` would be nice if linters could be configured to double-check. But probably not worth the time investment in the near term.